README.md

```diff
@@ -107,3 +107,66 @@ _Args(a=2, b=<Bird.puffin: 1>, c=None)
 $ uv run examples/example_1.py --a 2 --b lark --c /somewhere/to/go
 _Args(a=2, b=<Bird.lark: 2>, c=PosixPath('/somewhere/to/go'))
 ```
+
+## Pitfall: forward-references
+
+All types used in annotations _must_ be available at runtime.
+
+Specifically, when you call `argparcel.parse(Args)`, internally it relies upon
+`typings.get_type_hints(_Args)` working. It will not work if any of the type annotations
+for fields in `Args` can't be resolved at runtime. 
+
+A plausible scenario for this to fail is when using forward references.
+
+Forward references are used when:
+- annotating with a string
+- you have `from future import __annotations__` in the module
+- you're using Python 3.14 or later
```

Comment:
Plausibly mis-understandable as joined with ', and'

```suggestion
- annotating with a string, or
- you have `from future import __annotations__` in the module, or
- you're using Python 3.14 or later.
```

(I see you start the next sentence with "In any of these cases", but maybe this helps for top-to-bottom reading.)